public/index.html

```diff
@@ -5,12 +5,9 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <script src="https://cdn.nimiq-testnet.com/web-offline.js"></script>
+    <script src="https://cdn.nimiq-testnet.com/web-offline.js" defer></script>
```

Comment:
Is there a way to add `defer` to the `index.js` that webpack injects too?
Otherwise, I think, `index.js` will be executed before `web-offline.js`. Thus, if there is already a call to `Nimiq.something` at that point, it would fail.

Reply:
Yes, I'm running into this when merging this. So I suggest moving the Nimiq lib to the bottom directly above the build injection comment and removing the defer.

Reply:
But by using `defer` in the head the browser immediately starts downloading the file.

Reply:
Then we need to make the build js files also have defer. Maybe this can be integrated: https://www.npmjs.com/package/script-ext-html-webpack-plugin

Reply:
That plugin didn't work (html outcome ignored our index.html, maybe needed more configuration...). I wrote a tiny script instead to do that.